src/templates/static/metatrader/download.jsx

```diff
@@ -104,13 +104,15 @@ const DesktopDownloadBlock = ({
 );
 
 const MT5DesktopApp = ({ is_first_child, has_desktop_app }) => {
-    const txt_unsupported = it.L('The MT5 desktop app is not supported by macOS, Windows XP, Windows 2003, and Windows Vista.');
+    const txt_unsupported = it.L('The MT5 desktop app is not supported by Windows XP, Windows 2003, and Windows Vista.');
```

Comment:
we do support macOs, any reason as of why do we remove it?

Reply:
@mustofa-binary don't know why it was ommited, but as I understand, download option for mac should be added to binary now

Reply:
@mustofa-binary It seems that some months ago showing the download for mac removed, and now the QA asked for that.
--https://github.com/binary-com/binary-static/pull/5965/files